recipe/meta.yaml

```diff
@@ -24,19 +25,25 @@ requirements:
     - {{ compiler('rust') }}
     - cargo-bundle-licenses
   host:
-    - maturin >=0.13,<0.14
+    - maturin =0.13.7
```

Comment:
I wonder why it succeeded with `maturin 0.13.7` because it requires` maturin>=0.14,<0.15`  https://github.com/y-crdt/ypy/blob/09345f0aadcdbd67a02101e2dafa9163f2b32fda/pyproject.toml#L2

Reply:
The pins upstream might just be to avoid failures or unexpected behaviours. SO basically prevention. Which means it's highly possible it still works with slightly older versions.